KsApi/models/Comment.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+public struct Comment: Decodable, Equatable {
+  public var author: Author
+  public var body: String
+  public var id: String
+  public var uid: Int
+  public var replyCount: Int?
```

Comment:
`GraphComment` doesn't make `replyCount` optional, do we need to here?

Reply:
Oh yeah good point, will make it non-optional.